packages/react-keytips/src/components/Keytips/Keytips.test.tsx

```diff
@@ -0,0 +1,192 @@
+import * as React from 'react';
+import { render, screen, act } from '@testing-library/react';
+import { Keytips } from './Keytips';
+import { useKeytipRef } from '../..';
```

Comment:
I suggest being explicit by providing module from which you're importing. while this work now it wont work with ESM in future. also importing from barrel files === slower test execution